.gitignore

```diff
@@ -19,3 +19,5 @@ code/provision/.vagrant
 code/provision/playbooks/vault-password.txt
 
 code/provision/playbooks/.artifacts/
+
+code/docker/spark/spark-2\.1\.0-bin-hadoop2\.7\.tgz
```

Comment:
Do you want to exclude the entire code/docker/spark directory, in case you need to add in different spark/hadoop versions?